process.c

```diff
@@ -3090,21 +3090,7 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
  *
  *    Foo
  *
- *  On a Unix-like system, the shell is <tt>/bin/sh</tt>;
- *  otherwise the shell is determined by environment variable
- *  <tt>ENV['RUBYSHELL']</tt>, if defined, or <tt>ENV['COMSPEC']</tt> otherwise.
- *
- *  Except for the +COMSPEC+ case,
- *  the entire string +command_line+ is passed as an argument
- *  to {shell option -c}[https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/sh.html].
- *
- *  The shell performs normal shell expansion on the command line:
- *
- *    exec('echo C*')
- *
- *  Output:
- *
- *    CONTRIBUTING.md COPYING COPYING.ja
+ *  See {Execution Shell}[rdoc-ref:Process@Shell] for details about the shell.
```

Comment:
Oops, I missed that this link is incorrect. I fixed it in commit 154bd04ee228ee3bee173a479d397074268cd4d6.

Reply:
Three of these, right?

Reply:
Yep. All three were fixed in 154bd04ee228ee3bee173a479d397074268cd4d6.